src/components/BarChart.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <BarChart
+    v-bind="barChartProps"
+    :style="{
+      width: '15rem',
+      height: '15rem',
+      alignItems: 'center',
+    }"
+  />
+</template>
+
+<script>
+import { computed, defineComponent, ref, toRefs } from "vue";
+import { BarChart, useBarChart } from "vue-chart-3";
+import { Chart, registerables } from "chart.js";
+
+const dataChange = (data) => {
+  let arr = [];
+  data.forEach((element) =>
+    element > 5 ? arr.push(-element) : arr.push(element)
+  );
+  return arr;
+};
```

Comment:
```suggestion
    dataChange(data) {
     return data.map((element) => element > 5 ? -element : element);
    },
```
**Recommend-Request**: map 사용해서 단축이 가능할 것 같습니다.

Reply:
오 넹넹 감사합니당 바로 수정하겠습니당!